docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,23 +1,26 @@
 [[modules-remote-clusters]]
 == Remote clusters
 
-The _remote clusters_ functionality enables you to establish unidirectional
-connections to a remote cluster. Remote clusters are required for
+Set up _remote clusters_ to establish unidirectional
+connections required for
 <<xpack-ccr,{ccr}>> and <<modules-cross-cluster-search,{ccs}>>.
 
-Remote cluster connections work by configuring a remote cluster and connecting
-to a limited number of nodes in that remote cluster. There are two modes for
-remote cluster connections: <<sniff-mode,sniff mode>> and
-<<proxy-mode,proxy mode>>.
+You configure a remote cluster and connect
+to a limited number of nodes in that remote cluster using either
+<<sniff-mode,sniff mode>> or <<proxy-mode,proxy mode>>.
```

Comment:
I'd say "register" the remote cluster rather than "create" or "configure":

```suggestion
To register a remote cluster, the local cluster must connect to  nodes in the
remote cluster. You can do this using one of two connection modes:

* <<sniff-mode,sniff mode>>
* <<proxy-mode,proxy mode>>
```